utils.py

```diff
@@ -20,23 +17,19 @@
 import matplotlib.image as mpimg
 
 
-
-def take_screenshot():
-    screen = wx.ScreenDC()
-    bmp = wx.Bitmap(Screenshot.SRC_W, Screenshot.SRC_H)
-    mem = wx.MemoryDC(bmp)
-    mem.Blit(0, 0, Screenshot.SRC_W, Screenshot.SRC_H, screen, Screenshot.OFFSET_X, Screenshot.OFFSET_Y)
-    return bmp
-
-
-
 def prepare_image(img):
+    import wx
```

Comment:
ah right here it is: https://github.com/kevinhughes27/TensorKart/blob/master/utils.py#L182

When it is called from here its never a wx image so the `if` clause isn't used. I think this can be replaced with just `resize_image`